.circleci/config.yml

```diff
@@ -175,45 +183,49 @@ jobs:
       - store_test_results:
           path: _reports
 
-  e2e_testnet_leanhelix:
+  e2e_testnet:
     docker:
       - image: circleci/golang:1.12.9
+    environment:
+      BOOTSTRAP_URL: "https://boyar-testnet-bootstrap.s3-us-west-2.amazonaws.com/boyar/config.json"
```

Comment:
why this name? maybe a better name would be `VCHAINS_CONFIG_FILE` or some other name with a similar spirit?

Reply:
The bootstrap URL name is coming from the Nebula realms where this file location is called bootstrapUrl from the era in which we developed Nebula so I'm trying to stick to the same naming convention.